plugin-server/src/utils/retries.ts

```diff
@@ -0,0 +1,112 @@
+import { RetryError } from '@posthog/plugin-scaffold'
```

Comment:
The changes here are impossible to follow because we both moved a file and then refactored it in the same PR.

Please keep a clear commit history in cases like this or e.g. do the move in a follow-up PR as otherwise you make the reviewers job worse.